sandbox/controls/ListItems.qml

```diff
@@ -322,6 +322,19 @@ CExPynn1gWf9bx498P7/nzPcxEzGExhBdJGYihtAYQlO+tUZvqrPbqeudo5iJGEJjCE15a3VtodH3q2I
         badge.icon.isLetterIdenticon: true
     }
 
+    StatusListItem {
+        title: "List Item with Tags"
+        icon.isLetterIdenticon: true
+        tags: [
+            StatusListItemTag {
+                title: qsTr("Tag 1")
+                icon.isLetterIdenticon: true
+                onClick: {
```

Comment:
Please add a console.log or something or delete it if not needed :)

Reply:
removed